packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -0,0 +1,224 @@
+// @flow
+
+import * as React from "react";
+import ReactDOM from "react-dom";
+import {StyleSheet} from "aphrodite";
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import {getClickableBehavior} from "@khanacademy/wonder-blocks-core";
+import DropdownCore from "./dropdown-core.js";
+import type {Item, DropdownItem} from "../util/types.js";
+import ActionItem from "./action-item.js";
+import OptionItem from "./option-item.js";
+import DropdownAnchor from "./dropdown-anchor.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * The child function, that takes in state the state of the element and
+     * returns an Element which will become the opener element for the dropdown.
+     */
+    children: (state: ClickableState) => React.Element<any>,
```

Comment:
Not sure about naming this argument as `state`. When I read the example, I got a bit confused b/c I thought you were sharing React's `state`, but you're sharing the "DOM" state of the element instead, is that the case?.

Reply:
That's on me.  I think I called it `state` when I originally wrote `ClickableBehavior`.  Agree that the name could be be improved.